Saudox/tests/Feature/RotasTest.php

```diff
@@ -11,6 +11,7 @@ class RotasTest extends TestCase {
     // TODO: Ajustar esses testes depois. Aqui podemos definir o padrão das
     //      rotas que iremos utilizar nos demais testes.
     /* TODO: Aqui ainda falta colocar os testes de "novo alguma coisa"... */
+    /* TODO: Tirar esses testes de vez...? */
```

Comment:
## Acredito que sim, pois eles serão necessários apenas quendo for feito alguma refatoração nas rotas, que é algo bem raro.
### Nesse caso, deixa eles aí, só 'desliga' eles

Reply:
já tá desligadinho